code/modules/mob/living/simple_animal/slime/slime.dm

```diff
@@ -107,9 +111,13 @@
 	set_nutrition(700)
 
 /mob/living/simple_animal/slime/Destroy()
-	for (var/A in actions)
+	for(var/A in actions)
 		var/datum/action/AC = A
 		AC.Remove(src)
+	for(var/spawner in GLOB.mob_spawners)
+		LAZYREMOVE(GLOB.mob_spawners[spawner], src)
```

Comment:
Since this is added to the list by:
`LAZYADD(GLOB.mob_spawners["[master.real_name]'s slime"], M)`
Can't you remove by just removing from
`LAZYREMOVE(GLOB.mob_spawners["[master.real_name]'s slime"], M)`